global.d.ts

```diff
@@ -10,10 +10,7 @@ interface Camp {
   campStatus?: boolean
 }
 
-interface Role {
-  role: string
-  name: string
-}
+type Role = 'campCamper' | 'campAdmin' | 'globalAdmin' | 'all' | 'test'
```

Comment:
What’s “test”?

Reply:
It's a test role for demonstrating how the routes work. Let me take it off.